assets/sass/local.sass

```diff
@@ -27,17 +28,25 @@ header.govuk-header
   flex-grow: 1
   text-decoration: none
   padding-left: 0px
+  @include govuk-media-query($from: 0px)
+    margin-left: 58px
+  @include govuk-media-query($from: 360px)
+    margin-left: inherit
 
 .logo-text
   padding: 3px 15px 0 10px
-  border-right: solid white 1px
   margin-right: 15px
+  @include govuk-media-query($from: 0px)
+    border-right: none
+  @include govuk-media-query($from: 360px)
+    border-right: solid white 1px
 
 .user-block
   color: white
   float: right
   font-size: 19px
-  background-color: govuk-colour("blue")
+  background-color: #015ea5
```

Comment:
Bespoke colours like this are probably due to Eddie ignoring the Gov UK palette.  I'd suggest introducing a SASS variable for this colour.  That would make it easy to change its value later, and also makes it clear that all the places it is used are sharing the same value.  There are two occurrences in this file already.

Reply:
As suggested, have created a new file called palette.sass and defined the variable $govuk-blue for use in the local.sass
Have also updated the media queries from using actual px to $from: mobile and $from: desktop 
Lastly, have changed the padding in the ul and li so that links can all fit on one line when screen is in mobile size